### PR TITLE
Fix inverted condition during short segment check 

### PR DESCRIPTION
**Pull request by johnBartos**
- Causes audio to start behind live edge, delaying playback

JW7-4030

### CODE REVIEW

File: src/controller/audio-stream-controller.js [hunk 1]
```diff
@@ -268,7 +268,7 @@ class AudioStreamController extends EventHandler {
                 maxFragLookUpTolerance = 0;
               }
               // Prefer the next fragment if it's within tolerance
-              if (fragNext && fragmentWithinToleranceTest(fragNext) !== 0) {
+              if (fragNext && !fragmentWithinToleranceTest(fragNext)) {
```
Comment (robwalch) [inline src/controller/audio-stream-controller.js]: Don't `let` your `const`s get the best of you!

**Comment by msiddique** [comment]
89/92 tests passed. 3 test(s) failed. The 3 failed tests worked fine when I ran them manually.

**Comment by dannyfinks** [comment]
I looked more into some of these and they were legit failures. I will reject this ticket based on those issues.

**Comment by dannyfinks** [comment]
disregard my last comment, it appears that all the failures came from tests that were added as part of 7.9.1/7.9.2 and perhaps the PR here did not include some of those changes. I built the latest 7.10 where these pass

**Pull request merged**